src/julia-parser.scm

```diff
@@ -2360,6 +2360,18 @@
                        ((comprehension) `(braces ,@(cdr vex)))
                        (else   `(bracescat ,@(cdr vex))))))))
 
+          ((eqv? t #\⟨ )
+           (take-token s)
+           `(call anglebracket ,@(parse-call-arglist s #\⟩)))
```

Comment:
The way we'd usually do this is to parse it as `(anglebracket ...)`, so that it can be distinguished from the user writing `anglebracket(...)`. Then add a lowering rule in julia-syntax.scm to convert it to a call. See for example how `vect` is handled.

Reply:
okay